src/formats/onenote.ts

```diff
@@ -711,8 +741,12 @@ export class OneNoteImporter extends FormatImporter {
 			const outputPath = await this.fetchAttachment(progress, fileName, contentLocation);
 			if (outputPath) {
 				image.src = encodeURI(outputPath);
-				if (!image.alt || BASE64_REGEX.test(image.alt)) image.alt = 'Exported image';
-				else image.alt = image.alt.replace(/[\r\n]+/gm, '');
+				if (this.disableOCR || !image.alt || BASE64_REGEX.test(image.alt)) {
+					image.alt = 'Exported image';
+				} else {
+					// Sanitize OCR text to ensure valid markdown
+					image.alt = this.sanitizeOCRText(image.alt);
```

Comment:
Move the fallback option closer to the other default text.

```suggestion
					image.alt = this.sanitizeOCRText(image.alt) || 'Exported image';
```